src/components/trace/hat-graph-spacer.ts

```diff
@@ -31,15 +26,21 @@ export class HatGraphSpacer extends LitElement {
       :host {
         display: flex;
         flex-direction: column;
+        align-items: center;
+      }
+      svg {
+        width: var(--hat-graph-spacing);
+        height: calc(
+          var(--hat-graph-spacing) + var(--hat-graph-node-size) + 1px
+        );
       }
-      :host(.track) {
+      :host([track]) {
```

Comment:
Shouldn't this be registered as a property of the element too? For typing?

Reply:
Added jsdoc